ca_cdk_constructs/eks/imports/io/external_secrets/generators/_jsii/__init__.py

```diff
@@ -0,0 +1,27 @@
+import abc
```

Comment:
not sure what this file is, but I guess it was autogenerated.

Reply:
This was generated by the `cdk8s import --output ca_cdk_constructs/eks/imports` command.